### PR TITLE
Make base variables into custom properties

### LINKED ISSUE #574: Convert Base component to CSS modules
# Issue Description
To better align with modern CSS practices and modern CSS theming capabilities, we want to update this component to use CSS modules and if applicable, CSS custom properties.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Component styles are converted to use CSS modules. If applicable, convert SCSS variables to CSS custom properties.

## Current Behavior
Component styles use manual name spaced classes and SCSS variables.

### PR DESCRIPTION
**Pull request by mhemesath**
### Summary
Made font size, font family and base color into custom properties.

Resolves #574

### CODE REVIEW

File: packages/terra-base/src/Base.scss [hunk 1]
```diff
@@ -4,16 +4,16 @@
 // More info: https://css-tricks.com/box-sizing/#article-header-id-6
```
Comment (Matt-Butler) [inline packages/terra-base/src/Base.scss]: When we migrate to css modules, do we want to convert these from scss to css?

Comment (mhemesath) [inline packages/terra-base/src/Base.scss]: We can could flip this file to CSS, there is no reason it needs to be scss at this point.

**Comment by mhemesath** [comment]
One thing I'm not sure of is how this component will integrate into an MPages environment (once not in iframes). I think we should  try to identify strategies for mitigating the global html/body properties and push all others to a wrapper div we can scope to Terra components. Thoughts?

File: packages/terra-base/src/Base.scss (hunk 1 above)
Comment (Matt-Butler) [inline packages/terra-base/src/Base.scss]: +1 I added this to our discussion on Friday. I don't want this to block your review.

**Pull request merged**